rslib/src/scheduler/fsrs/simulator.rs

```diff
@@ -61,3 +74,20 @@ impl Collection {
         })
     }
 }
+
+impl Card {
+    fn convert(card: Card, col: &mut Collection) -> fsrs::Card {
+        if let Some(state) = card.memory_state {
+            let due = card.original_or_current_due();
+            let relative_due = due - col.timing_today().unwrap().days_elapsed as i32;
```

Comment:
It would be better to get the timing once, and pass it into convert(), instead of fetching it for every card.

Reply:
OK. I will update it soon.